### ISSUE TITLE
Error finding offsets

### ISSUE THREAD

**Issue by SadismoZ** [pwn20wndstuff/Undecimus#2052]
Iphone 6 plus ios version 12.1.3 

Unc0ver 5.0.1/5.0.2

Errors

![22670B3C-AC54-4E95-AD58-CE4C5613AE41](https://user-images.githubusercontent.com/66485821/84371304-bb7ce980-abd9-11ea-8604-594d279fc16d.png)

And

![4C28A6D4-A85E-47DA-B1E8-D7D381582743](https://user-images.githubusercontent.com/66485821/84371506-0a2a8380-abda-11ea-82a4-9e9b8845927b.png)

**Comment by pbgbottle** [comment]
Yep same on all my iOS 12.4 devices. Right when the ad pops up it causes it to fail.

**Comment by SadismoZ** [comment]
> Yep same on all my iOS 12.4 devices. Right when the ad pops up it causes it to fail.

Same but i fixed this you must dowlnload an older version of unc0ver 

Follow this guide and download unc0ver 4.3.1

https://youtu.be/hy1JbWBbJ3M

**Comment by Hamlock-maneuverr** [comment]
Yes.  unc0ver 5 and up, is meant for ios 13.
